### ISSUE TITLE
Vestibular VR

### ISSUE THREAD

**Issue by glopesdev** [neurogears/vestibular-vr#26]
Meta-issue to track ongoing development tasks.

- [ ] #21 
- [ ] #25 
- [x] #47
- [x] #48 
- [x] #49

**Comment by EleonoraAmbrad** [comment]
In the current implementation of the VR algorithm (i.e. halt settings etc), I am not sure if thre is a way to modify rotation of drum. The idea would be to, instead of halting visual flow as it is in the linear corridor, start rotating an otherwise fixed drum in closed loop with the motor to introduce a visuo-vestibular mismatch. I think this was our idea behind the Halt Gain parameter, but i cannot get it to do this.

**Comment by RoboDoig** [comment]
Yes, the halt gain parameter should allow you to do this. Does any kind of halting work with the drum?
